includes/admin/post-types/meta-boxes/class.llms.meta.box.course.options.php

```diff
@@ -356,7 +357,10 @@ public function get_fields() {
 			),
 		);
 
-		if ( function_exists( 'register_block_type' ) && llms_blocks_is_post_migrated( $this->post->ID ) ) {
+		if (
+			( ! class_exists( 'Classic_Editor' ) && 'auto-draft' === get_post_status( $this->post->ID ) ) ||
```

Comment:
what if the classic editor exists (plugin installed and active) but the current user uses the block editor by default?
have you checked this?

Reply:
Thanks @eri-trabiccolo 

Didn't know about the switching aspect of the editors. Just pushed a fix, and now it's working fine.